js/index.js

```diff
@@ -1 +1,52 @@
 // Your code here
+var num =0;
+function diceroll(){
```

Comment:
This function is too big! A function should do one small job and you combine a bunch together to do a big job like this.